libevmasm/AssemblyItem.h

```diff
@@ -105,6 +108,25 @@ class AssemblyItem
 		m_debugData{langutil::DebugData::create()}
 	{}
 
+	static AssemblyItem functionCall(uint16_t _functionID, uint8_t _args, uint8_t _rets, langutil::DebugData::ConstPtr _debugData = langutil::DebugData::create())
+	{
+		// TODO: Make this constructor this way that it's impossible to create it without setting functions signature.
+		// It can be done by template constructor with Instruction as template parameter i.e. Same for JumpF below.
```

Comment:
I see you still left `0` as `data` there because we don't have a constructor that would let you omit it. But I guess without data `RetF` does not even need its own assembly item could be simply an `Operation`?

In any case it not overly important now, let's merge the PR, but we should consider refactoring it at some later point.